### PR TITLE
Merge Branch jj 0.6.2.4 into Master

### PR DESCRIPTION
**Pull request by snappy-sachin**
# Changes proposed in this pull request

- Bump up the versions Store 1.5.4, Spark 2.0.3-3 and Jobserver 0.6.3-3
- Change to a new version naming policy.
- [SNAPPYDATA] explicitly exclude spark-unsafe

# Patch testing
- Pre-checkin yet to be run

# Other PRs

- SnappyData : https://github.com/SnappyDataInc/snappydata/pull/598
- Spark : https://github.com/SnappyDataInc/spark/pull/49
- Store: https://github.com/SnappyDataInc/snappy-store/pull/185
- AQP: https://github.com/SnappyDataInc/snappy-aqp/pull/146

### CODE REVIEW

**Review by sumwale — APPROVED** [review]
LGTM. Check the version with @ashetkar if that is what should be for master.

**Comment by sumwale** [comment]
In the merge message, add [SNAPPYDATA] in the heading at the start which is our convention for external components spark-jobserver and spark commits.

File: build.gradle [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
 apply plugin: 'scala'
 
-version = '0.6.3-2'
+version = '0.6.2.3'
```
Comment (ashetkar) [inline build.gradle:20]: The master branch 'snappydata' already has this change. I don't understand why is it showing here in the diff.
In fact, I see all the three checkins already in this master branch. May be, this PR is not needed or needs to be re-created?

Comment (sumwale) [reply build.gradle:20]: @snappy-sachin Try "git fetch origin" and then "git merge origin/snappydata" into branch-jj-0.6.2.4 again.

Comment (ashetkar) [reply build.gradle:20]: Here's what I got from GitHub folks:
> The compare view works very much like the following Git command:
`git diff snappydata...branch-jj-0.6.2.4`
> This command lets you view the changes on branch-jj-0.6.2.4, starting at a common ancestor of both snappydata and branch-jj-0.6.2.4.
> In other words, it gives you a list of changes on branch-jj-0.6.2.4 from the point you branched off from snappydata. Whether or not something happened on snappydata in the meantime is not relevant.
`git diff snappydata..branch-jj-0.6.2.4`
> The command above, though, will show you a diff between the two latest commits in those branches (notice that there are two dots, not three)

Comment (sumwale) [reply build.gradle:20]: @snappy-sachin As mentioned by @ashetkar the diff command above shows none of the above changes, so this PR can be closed and the jj branch deleted.

**Comment by sumwale** [comment]
Closing this PR as per last comments.

**Pull request closed without merging**